### ISSUE TITLE
DAC 20 - Medium Priority WCAG Level AA - Colour Contrast [Non-Text] 07

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#20]
<img width="402" alt="DAC_Colour_Contrast_Non_Text_07" src="https://user-images.githubusercontent.com/75850106/155561374-63447524-4837-46ce-9511-16e109aa1fcb.png">

### Issue

There are multiple green dots within the ‘Income deprivation and Moran's I in rural and urban local authorities, 2019’ diagram. The green colour used does not meet the minimum contrast of 3:1 against adjacent colours. Low vision users may have difficulty seeing the green dots.

Foreground: #A8BD3A 
Background: #FFFFFF
The contrast ratio is: 2.1:1 
1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

<img width="599" alt="DAC 20 Current code ref" src="https://user-images.githubusercontent.com/75850106/155561502-0446f2d8-3d4a-4424-a73a-4c4418db658f.png">

**Low vision comments:**
“The green dots used on the chart representing Income deprivation clustering is lower in rural areas, fails non-text colour contrast at an approximate ratio of 2.1:1, ideally this should be above 3:1”

### Suggested Solution
Ensure that that non-text colour contrasts meet the minimum ratio of 3:1.
